apps: add deployment option for ingress and nodeport (#1431)
elastisys/compliantkubernetes-apps: WIP-CHANGELOG.md

- Compaction for logs stored directly in object store by Fluentd
- New 'Log review overview' Opensearch dashboard
- Added falco rules to ignore redis operator related alerts.

> - Added the option to set ingress Nginx as deployment.

Reviewer: ```suggestion
- Added option to deploy ingress-nginx as a deployment.
```
